### DISCUSSION TITLE
Remove buttons for person receiving my links

### DISCUSSION THREAD

**Discussion by chrish112** [PrivateBin/PrivateBin#1368] [Q&A] [answered]
### Describe the problem/question

Hi Everyone,
I'm trying to find a way to make it so that the button options are only for the creators of the paste but not for the receiver of the paste. The idea is to make it as simple as possible for the person receiving, as if someone can mess it up they will mess it up. I have been using this project to send information to customers but they seem to think this is a project that allows them to directly upload content to me or they click new to send the information back to me through the link. 

The goal is to only have button options for the creator of the link, the receiver can only view the download attachment or view the text file. I can remove many options on the webpage but removing the "NEW" button removes it from the creator and receiver's view.

I want to know if someone knows of an easy way to do this as the next idea for me to test is if I create a 2nd privatebin website, mount the data folder from the previous site as a shared location, have the first site generate links towards the 2nd website.
I really don't know right now how this will function but this is the idea.

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

Question and steps in first body

### Server OS

Ubuntu 22 Server

### Webserver

Apache2

### PrivateBin version

v1.7.3

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by rugk** [comment, marked as answer]
So you basically don't want receivers to submit (upload) pastes, but only "you"? So you basically want a "read-only view" for pastes. See this FAQ for an answer how this may be done: https://github.com/PrivateBin/PrivateBin/wiki/FAQ#how-can-i-link-to-a-read-only-mode-where-users-cant-submit-pastes
